src/Stratis.Bitcoin.IntegrationTests/ConsensusManagerTests.cs

```diff
@@ -609,6 +610,42 @@ public void ConsensusManager_Fork_Occurs_When_Same_Coins_Are_Staked_On_Different
             }
         }
 
+        [Fact]
+        public void ConsensusManager_Block_That_Failed_Partial_Validation_Is_Rejected()
+        {
+            using (NodeBuilder builder = NodeBuilder.Create(this))
+            {
+                var network = new StratisConsensusOptionsOverrideTest();
+
+                // MinerA requires a physical wallet to stake with.
+                var minerA = builder.CreateStratisPosNode(network, "minerA").OverrideDateTimeProvider().WithWallet().Start();
```

Comment:
I don't think that you need to override the date time provider as you aren't staking (but just check if I'm right)